src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -60,3 +61,58 @@ def test_ipa_trusts__lookup_group_without_sid(ipa: IPA, trusted: GenericADProvid
     status = ipa.sssctl.domain_status(trusted.domain, online=True)
     assert "online status: offline" not in status.stdout.lower(), "AD domain went offline!"
     assert "online status: online" in status.stdout.lower(), "AD domain was not online!"
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopologyGroup.AnyIPATrust)
+def test_ipa_trusts__ipa_server_trusted_user_lookup(ipa: IPA, trusted: IPA):
```

Comment:
I think a better name would be 

test_ipa_trusts__on_the_ipa_master_lookup_a_trusted_user"

Just to make it a little clearer that the test doesn't use a client.

Reply:
I changed this to `test_ipa_trusts__ipa_master_lookup_trusted_user`, just to be a bit more concise.